init.ps1

```diff
@@ -94,7 +94,7 @@ Write-Host "Adding Windows hosts file entries..." -ForegroundColor Green
 Add-HostsEntry $cmHost
 Add-HostsEntry $cdHost
 Add-HostsEntry $idHost
-Add-HostsEntry $renderingHost
+Add-HostsEntry $mvprenderingHost
```

Comment:
This fails, since 4 lines above we have this:

```
$renderingHost = "mvp.sc.localhost"
```

This variable needs to be changed as well.